Flaky test: org.keycloak.testsuite.actions.RequiredActionPriorityTest#executeRequiredActionWithCustomPriorityAppliesSamePriorityToSessionAndUserActions

## org.keycloak.testsuite.actions.RequiredActionPriorityTest#executeRequiredActionWithCustomPriorityAppliesSamePriorityToSessionAndUserActions
[Keycloak CI - Base IT (1)](https://github.com/keycloak/keycloak/actions/runs/13435213279/job/37536106603) / [Pull Request #37379](https://github.com/keycloak/keycloak/pull/37379)

```
java.lang.AssertionError: Event expected
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertNotNull(Assert.java:713)
	at org.keycloak.testsuite.AssertEvents.poll(AssertEvents.java:91)
...
```

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/RequiredActionPriorityTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/RequiredActionPriorityTest.java
@@ -349,6 +349,8 @@ public class RequiredActionPriorityTest extends AbstractTestRealmKeycloakTest {
         /* NOTE: we don't configure VERIFY_PROFILE on the user - it's set on the session because the profile is incomplete */
         enableRequiredActionForUser(RequiredAction.TERMS_AND_CONDITIONS);
 
+        // make sure no action is requested
+        oauth.kcAction(null);
         // Get a password reset link
         loginPage.open();
         loginPage.assertCurrent(TEST_REALM_NAME);

